solidity/contracts/wrapperproxy/test/testRBTCWrapper.js

```diff
@@ -0,0 +1,62 @@
+    });
+
+    it('verifies that users could remove liquidity to burn pool token and then get RBTC', async () => {
+        console.log('The account address:', account);
+        console.log('The amount of RBTC of the account:', await web3.eth.getBalance(account));
+        console.log('The amount of pool token of the account:', await poolToken.balanceOf(account));
+        await poolToken.approve(RBTCWrapperProxy.address, await web3.utils.toBN(1e16), {from:account});
+        console.log('The allowance of pool token that the RBTCWrapperProxy smart contract could spend:', await poolToken.allowance(account, RBTCWrapperProxy.address));
+        await rbtcWrapperProxy.removeLiquidity(web3.utils.toBN(1e16), 1, {from:account, to:RBTCWrapperProxy.address});
+        console.log('The amount of RBTC of the account after burning 0.01 pool token:', await web3.eth.getBalance(account));
+        console.log('The amount of pool token of the account after burning 0.01 pool token:', await poolToken.balanceOf(account));
+    });
+    
+    it('verifies that users could send RBTC and then swap it to DoC', async () => {
+        console.log('The account address:', account);
+        console.log('The amount of RBTC of the account:', await web3.eth.getBalance(account));
+        console.log('The amount of DoC of the account:', await docToken.balanceOf(account));
+        var path = await sovrynSwapNetwork.conversionPath(IERC20Token(wrbtcAddress), IERC20Token(docTokenAddress));
+        await rbtcWrapperProxy.convertByPath(path, web3.utils.toBN(1e16), 1, {from:account, to:RBTCWrapperProxy.address, value:1e16});
+        console.log('The amount of RBTC of the account after sending 0.01 RBTC:', await web3.eth.getBalance(account));
+        console.log('The amount of DoC of the account after sending 0.01 RBTC:', await docToken.balanceOf(account));
+    });
+
+    it('verifies that users could send DoC and then swap it to RBTC', async () => {
+        console.log('The account address:', account);
+        console.log('The amount of RBTC of the account:', await web3.eth.getBalance(account));
+        console.log('The amount of DoC of the account:', await docToken.balanceOf(account));
+        await docToken.approve(RBTCWrapperProxy.address, await web3.utils.toBN(1e20), {from:account});
+        console.log('The allowance of DoC that the RBTCWrapperProxy smart contract could spend:', await docToken.allowance(account, RBTCWrapperProxy.address));
+        var path = await sovrynSwapNetwork.conversionPath(IERC20Token(docTokenAddress), IERC20Token(wrbtcAddress));
+        await rbtcWrapperProxy.convertByPath(path, web3.utils.toBN(1e20), 1, {from:account, to:RBTCWrapperProxy.address});
+        console.log('The amount of RBTC of the account after sending 100 DoC:', await web3.eth.getBalance(account));
+        console.log('The amount of DoC of the account after sending 100 DoC:', await docToken.balanceOf(account));
+    });
+
```

Comment:
negative tests are missing. make sure the transactions fail when they're supposed to fail.